mlir/include/mlir/Dialect/Affine/IR/AffineOps.td

```diff
@@ -671,7 +671,7 @@ def AffineParallelOp : Affine_Op<"parallel",
      I32ElementsAttr:$lowerBoundsGroups,
      AffineMapAttr:$upperBoundsMap,
      I32ElementsAttr:$upperBoundsGroups,
-     I64SmallVectorArrayAttr:$steps,
+     I64SmallVectorArrayAttr:$step,
```

Comment:
well i did this because i called the interface method `getSteps` which would conflict with the generated getter for this operand. probably a bad idea but not sure what else to do (couldn't come up with better naames)

Reply:
Maybe make it `getLoopSteps` in the interface?

Reply:
works for me. thanks :)